docs/oss/pycharm-to-deepnote.md

```
@@ -0,0 +1,981 @@
+- Productionize in PyCharm
+
+<!-- IMAGE: Diagram showing PyCharm vs Deepnote use cases -->
+<!-- FILE: pycharm-vs-deepnote-usecases.png -->
+<!-- CAPTION: When to use PyCharm vs Deepnote -->
+
+## Key Differences
+
+### Development Paradigm
+
+**PyCharm (Script-based):**
+```python
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)